### ISSUE TITLE
Webextension building woes

### ISSUE THREAD

**Issue by dmyates** [kickscondor/fraidycat#11]
To get the webextension to build, I had to do the following:

1. Replace the local filepath for your router fork with a Github URL, as per the comment in https://github.com/kickscondor/fraidycat/pull/10
2. Replace all instances of `./js` with `./src/js` in `src/index.html` (I have a feeling this would break dat building).
3. Add the following files to `src/fonts/`:

```
overpass-400.woff2
overpass-400x.woff2
overpass-700.woff2
overpass-700x.woff2
```

With that done, the extension gives an `Uncaught (in promise)` error when it's loaded (in Chrome 78.0.3904.87) but appears to sort of work, bar a few broken images and an intolerance for multiple time categories (e.g. if I add a monthly, add a weekly and then switch back to the monthly, most of the interface disappears). I'm not sure if that's related to the version of router still being wrong or if it's a separate bug.

Otherwise this is really cool and I'm glad it's finally (almost) out!

**Comment by kickscondor** [comment]
Ok, amazing - thank you once again for all of this!

1. This is being addressed by 304ff74. Let me know if it's still a problem.
2. Very strange - I have some things to do, so I will return to try to fix this in two hours.
3. I've removed Overpass from the CSS in 7cfe40f, it isn't used any longer. (I'm sorry you had to track this down!)

The "uncaught (in promise)" is expected - this is a problem that appears when the background script is attempting to `sendMessage` to a tab that doesn't exist. It doesn't hinder usage of the extension, so it's lower on my priorities for now.

**Comment by dmyates** [comment]
Awesome, looks like it's all working now. Thanks! I'll report anything else I run into.
